goldenmaster/Plugins/TbSimple/Source/TbSimple/Private/Generated/OLink/TbSimpleSimpleArrayInterfaceOLinkClient.cpp

```diff
@@ -279,6 +457,44 @@ TArray<float> UTbSimpleSimpleArrayInterfaceOLinkClient::FuncFloat_Implementation
 	return Promise.GetFuture().Get();
 }
 
+TArray<float> UTbSimpleSimpleArrayInterfaceOLinkClient::FuncFloat32_Implementation(const TArray<float>& ParamFloat32)
+{
+	if (!m_sink->IsReady())
+	{
+		UE_LOG(LogTemp, Warning, TEXT("%s has no node"), UTF8_TO_TCHAR(m_sink->olinkObjectName().c_str()));
```

Comment:
Is there an ability to turn logging off?

Reply:
Not yet, created issue for this https://github.com/apigear-io/template-unreal/issues/33